packages/core/src/layout/HomepageTimer/HomepageTimer.tsx

```diff
@@ -39,18 +39,28 @@ function getTimes(configApi: ConfigApi) {
 
   for (const clock of clockConfigs) {
     if (clock.has('label') && clock.has('timezone')) {
+      let label = clock.getString('label');
+
       const options = {
         timeZone: clock.getString('timezone'),
         ...timeFormat,
       };
 
-      const time = d.toLocaleTimeString(lang, options);
-      const label = clock.getString('label');
+      try {
+        new Date().toLocaleString(lang, options);
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `The timezone ${options.timeZone} is invalid. Defaulting to America/Los Angeles`,
+        );
+        options.timeZone = 'America/Los_Angeles';
+        label = 'Los Angeles';
```

Comment:
```suggestion
          `The timezone ${options.timeZone} is invalid. Defaulting to Greenwich Mean Time (GMT).`,
        );
        options.timeZone = 'GMT';
        label = 'GMT';
```

Thanks to @OrkoHunter we know [where in the world the project folks are](https://twitter.com/SpotifyEng/status/1341375405446598656), and there are few Los Angelinos.  Maybe this should just default to GMT so as not to infer any West Coast/East Coast US preference?  (says the guy from the US East Coast 😉 )

Possible suggestion?

Reply:
+1 for GMT 😄

Reply:
GMT it is! I was toying with the idea of defaulting to New Zealand instead of LA, but this feels like a happy medium!